src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -858,7 +857,7 @@ protected override void Dispose(bool disposing)
             await props.ActiveDebugProfile.SetValueAsync(profileName).ConfigureAwait(true);
         }
 
-        internal async Task<string> GetLaunchSettingsFilePathAsync()
+        internal async Task<string> GetLaunchSettingsFilePathNoCacheAsync()
         {
             // NOTE: To reduce behavior changes, we currently cache the folder that we get from
```

Comment:
The comment is in the wrong spot - this method doesn't cache itself, but the result of this method is cached. I'll move the comment in a future PR as I didn't change that.